wizard/views/splash.py

```diff
@@ -0,0 +1,22 @@
+from state import CurrentStep
+import streamlit as st
+from views.side_buttons import set_form_step
+
+
+def render_splash():
+    st.text("Welcome to the Croissant Wizard, would you like to load an existing croissant file, or create a new one?")
+    col1, col2, col3 = st.columns([1,1,8], gap="small")
```

Comment:
Do you use VS Code? Can you use a Python formatter? `[1,1,8]` should be `[1, 1, 8]` and all files should have an EOF line return. Otherwise, it will create noise in the diff when someone updates the code afterwards.